src/main/java/ru/yandex/qatools/fsm/impl/YatomataImpl.java

```diff
@@ -15,7 +15,7 @@
 /**
  * @author Ilya Sadykov
  */
-public class YatomataImpl<T> implements Yatomata<T> {
+class YatomataImpl<T> implements Yatomata<T> {
```

Comment:
Sorry for inconvenience. If this class is actually a part of a public contract, I'll make another request to turn it back.